platform/kube/controller.go

```diff
@@ -579,10 +579,10 @@ func (c *Controller) HostInstances(addrs map[string]bool) []*model.ServiceInstan
 }
 
 const (
-	// IstioServiceAccountPrefix is always the prefix for Istio service accounts.
-	IstioServiceAccountPrefix = "istio:"
-	// LocalDomain is the domain name Istio service account uses for internal traffic.
-	LocalDomain = "cluster.local"
+	// the URI scheme used to encode a Kubernetes service account
+	uriScheme = "spiffe"
+	// the domain name Istio service account uses for internal traffic
+	localDomain = "cluster.local"
```

Comment:
We are following the SPIFFE standard which is for the cert SAN. So it makes more sense to be a URI path format. The two formats are 1-1 mapped, so I won't see a problem.